Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -911,15 +978,19 @@ def list_incidents_command(client: AzureSentinelClient, args, is_fetch_incidents
         A CommandResult object with the array of incidents as output.
     """
     filter_expression = args.get("filter")
-    limit = min(DEFAULT_LIMIT, int(args.get("limit")))
     next_link = args.get("next_link", "")
+    limit = min(arg_to_number(args.get("limit")) or DEFAULT_LIMIT, COMMAND_MAX_LIMIT)
```

Comment:
```suggestion
    limit = min(arg_to_number(args.get("limit")), COMMAND_MAX_LIMIT)
```
if we have the args from the command - we have default value in the yml. If from the fetch, we will always have value from the fetch_incidents method. Hence,

Reply:
This change will raise mypy error: 
Type "int | None" is not assignable to type "int" since min needs two int and arg_to_number return value is int|None

Reply:
The mypy error is because arg_to_number return None | int and min method want to get an int - so we leave it as is and didn't change it after a discussion.